tests/CppUTestExt/IEEE754PluginTest_c.c

```diff
@@ -25,6 +25,10 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <CppUTest/CppUTestConfig.h>
+
+#ifdef CPPUTEST_HAVE_FENV
```

Comment:
I probably shouldn't be using INFINITY in this file at all. Any idea how to rewrite the fucntion without INFINITY?